src/main/java/com/powsybl/openloadflow/ac/ReactiveLimitsOuterLoop.java

```diff
@@ -69,7 +69,7 @@ private void switchPvPq(LfBus bus, EquationSystem equationSystem, VariableSet va
 
     private void updateControlledBus(LfBus controlledBus, EquationSystem equationSystem, VariableSet variableSet) {
         // clean reactive power distribution equations
-        controlledBus.getControllerBuses().forEach(b -> equationSystem.removeEquation(b.getNum(), EquationType.ZERO_Q));
```

Comment:
I think that you should keep the type ````ZERO_Q````. Shunts and loads connected to generators are minor in real cases and we understand well the type ````ZERO_Q````.

Reply:
Done